packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -391,15 +404,19 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             ClaimData storage claim = claimData[challengeIndex];
 
             // Ignore false claims
-            if (!claim.countered) {
-                countered = true;
+            if (claim.counteredBy == address(0)) {
+                countered = msg.sender;
                 break;
             }
         }
 
+        // If the parent was not successfully countered, pay out the parent's bond to the claimant.
```

Comment:
this is good. I just want to explicitly call out that in the case there's a counter, we indeed want to reward the parent's bond to the _leftmost_ claimant, and not just any.